src/Pact/LSP/Server.hs

```diff
@@ -57,21 +57,22 @@ runWith i o l = do
       , hoverRequestHandler
       , completionRequestHandler
       ]
-    
+
     forward :: LanguageContextEnv ServerConfig -> HandlerM a -> IO a
     forward env handler =
         runLspT env $ do
           res <- runHandlerM handler
           case res of
             Left errMsg -> do
               -- send user notification for failure
-              sendNotification J.SWindowLogMessage J.LogMessageParams{_xtype = J.MtError, _message = T.pack (show errMsg)}
+              -- sendNotification Method_WindowLogMessage J.LogMessageParams{_type_ = J.MtError, _message = T.pack (show errMsg)}
```

Comment:
Shall this be removed entirely or uncommented or `TODO`ed?

Reply:
whup, this was a leftover. TY